[2021-05-15 10:15:51,606] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: etl.loading_data_to_s3 2021-05-15T11:00:00+00:00 [queued]>
[2021-05-15 10:15:51,611] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: etl.loading_data_to_s3 2021-05-15T11:00:00+00:00 [queued]>
[2021-05-15 10:15:51,611] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-15 10:15:51,611] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-05-15 10:15:51,611] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-15 10:15:51,616] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): loading_data_to_s3> on 2021-05-15T11:00:00+00:00
[2021-05-15 10:15:51,619] {standard_task_runner.py:52} INFO - Started process 27316 to run task
[2021-05-15 10:15:51,626] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'etl', 'loading_data_to_s3', '2021-05-15T11:00:00+00:00', '--job-id', '675', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/var/folders/_f/vg3x3jmj23bcz6s88l7v6fb40000gn/T/tmpdlkeq4qx', '--error-file', '/var/folders/_f/vg3x3jmj23bcz6s88l7v6fb40000gn/T/tmp5gzz_46h']
[2021-05-15 10:15:51,627] {standard_task_runner.py:77} INFO - Job 675: Subtask loading_data_to_s3
[2021-05-15 10:15:51,662] {logging_mixin.py:104} INFO - Running <TaskInstance: etl.loading_data_to_s3 2021-05-15T11:00:00+00:00 [running]> on host 27.2.168.192.in-addr.arpa
[2021-05-15 10:15:51,687] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=etl
AIRFLOW_CTX_TASK_ID=loading_data_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2021-05-15T11:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-15T11:00:00+00:00
[2021-05-15 10:15:52,953] {logging_mixin.py:104} INFO - Fetching interval list
[2021-05-15 10:15:52,955] {logging_mixin.py:104} INFO - Inverval list built
[2021-05-15 10:15:52,956] {logging_mixin.py:104} INFO - Processing data
[2021-05-15 10:15:52,956] {logging_mixin.py:104} INFO - df initialized
[2021-05-15 10:15:52,958] {logging_mixin.py:104} INFO - (array([1621080952000, 1621082752000]), 1)
[2021-05-15 10:15:52,959] {logging_mixin.py:104} INFO - (array([1621080952000, 1621082752000]), 2)
[2021-05-15 10:15:52,959] {logging_mixin.py:104} INFO - (array([1621080952000, 1621082752000]), 3)
[2021-05-15 10:15:52,960] {logging_mixin.py:104} INFO - ThreadPoolExecutor initialized
[2021-05-15 10:15:52,960] {logging_mixin.py:104} INFO - running function for: (array([1621080952000, 1621082752000]), 1)
[2021-05-15 10:15:52,961] {logging_mixin.py:104} INFO - https://cdn.iqoption.com/api/quotes-history/quotes/3.0?to=1621082752000&from=1621080952000&active_id=1&only_round=true&_key=1614903660000
[2021-05-15 10:15:52,963] {logging_mixin.py:104} INFO - submitted
[2021-05-15 10:15:52,963] {logging_mixin.py:104} INFO - running function for: (array([1621080952000, 1621082752000]), 2)
[2021-05-15 10:15:52,964] {logging_mixin.py:104} INFO - https://cdn.iqoption.com/api/quotes-history/quotes/3.0?to=1621082752000&from=1621080952000&active_id=2&only_round=true&_key=1614903660000
[2021-05-15 10:15:52,964] {logging_mixin.py:104} INFO - submitted
[2021-05-15 10:15:52,965] {logging_mixin.py:104} INFO - running function for: (array([1621080952000, 1621082752000]), 3)
[2021-05-15 10:15:52,965] {logging_mixin.py:104} INFO - https://cdn.iqoption.com/api/quotes-history/quotes/3.0?to=1621082752000&from=1621080952000&active_id=3&only_round=true&_key=1614903660000
[2021-05-15 10:15:52,966] {logging_mixin.py:104} INFO - submitted
[2021-05-15 10:15:56,793] {python.py:118} INFO - Done. Returned value was: None
[2021-05-15 10:15:56,800] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=etl, task_id=loading_data_to_s3, execution_date=20210515T110000, start_date=20210515T131551, end_date=20210515T131556
[2021-05-15 10:15:56,846] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-15 10:15:56,862] {local_task_job.py:146} INFO - Task exited with return code 0
